specification/sections/02-providers.md

```diff
@@ -250,6 +253,15 @@ class MyProvider implements Provider, AutoDisposable {
   }
 ```
 
+#### Requirement 2.5.2
+
+> After a provider's shutdown function has terminated successfully, the provider's state **MUST** revert to its uninitialized state.
```

Comment:
The non-normative section says "should," while the spec says" must".

Reply:
Good call, fixed.